### PR TITLE
Mesoshapes County Import - Round 2.1

### PR DESCRIPTION
**Pull request by stepps00**
This PR includes new admin2 (`county`) features for:

- Benin
- Bosnia and Herzegovina
- Burundi
- Central African Republic
- Djibouti
- Ghana
- Sierra Leone
- Syria
- Vietnam
- Zambia

Several of the countries in this PR currently have `county` features in Who's On First; updating current records occurred in several countries. Deprecating was needed for three existing features - two in Syria and one in Zambia. This will require PIP-ing features.

### CODE REVIEW

**Comment by stepps00** [comment]
Part of #558.

File: data/890/423/389/890423389-alt-quattroshapes.geojson [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "wof:geomhash":"ff5d56f5bb6c04c3b3710e325b8ff714",
+    "wof:id":890423389
+},
+  "bbox": [
+    107.24349,
+    10.80119,
+    107.24349,
+    10.80119
+],
+  "geometry": {"coordinates":[107.24348999999999,10.80119],"type":"Point"}
```
Comment (nvkelso) [inline data/890/423/389/890423389-alt-quattroshapes.geojson]: Nicely done!

File: data/110/877/907/1/1108779071.geojson [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    "lbl:latitude":8.34437,
+    "lbl:longitude":-13.133197,
+    "meso:adm1_en":"Western",
+    "meso:dissme":"WesternWestern Area",
+    "meso:gadm_adm2":"Western Rural",
+    "meso:gaul_adm2":"Western Area Rur",
+    "meso:hasc_id":"SL.WE.WR",
+    "meso:mps_x":-13.133197,
+    "meso:mps_y":8.34437,
+    "meso:name_en":"Western Area",
```
Comment (nvkelso) [inline data/110/877/907/1/1108779071.geojson]: This area around the Sierra Leon capital Freetown should probably be split in 2 for "Rural" and "Urban". 

- https://en.wikipedia.org/wiki/Districts_of_Sierra_Leone
- http://www.statoids.com/ysl.html

**Comment by stepps00** [comment]
Regarding https://github.com/whosonfirst-data/whosonfirst-data/pull/559#pullrequestreview-13642091: https://github.com/whosonfirst-data/whosonfirst-data/issues/593 has been filed and will fix the count discrepancy in Zambia.

**Comment by stepps00** [comment]
Regarding https://github.com/whosonfirst-data/whosonfirst-data/pull/559#pullrequestreview-13652356: The discrepancy (77 vs 87) is due to 9 `alt-geom` features in the commit. This still leaves us with a one feature discrepancy when comparing the mesoshapes import to Statoids and wikipedia, though, so https://github.com/whosonfirst-data/whosonfirst-data/issues/594 was opened to address the issue. 

The feature "Cove" is in the import features twice, but one feature covers the "city area" and one covers the "region area". Once imported, either merge into a multi-part feature or dissolve the smaller "city area" geometry into a surrounding feature.

File: data/110/875/957/9/1108759579.geojson [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    "meso:gadm_adm2":"Nd\u00e9l\u00e9",
+    "meso:gaul_adm2":"Nd\u00e9l\u00e9",
+    "meso:hasc_id":"CF.BB.ND",
+    "meso:mps_x":21.156755,
+    "meso:mps_y":8.780615,
+    "meso:name_en":"Ndele",
+    "meso:name_loc":"Nd\u00e9l\u00e9",
+    "meso:source":"AOTM",
+    "name:und_x_variant":[
+        "Nd\u00e9l\u00e9"
```
Comment (stepps00) [inline data/110/875/957/9/1108759579.geojson]: Opened https://github.com/whosonfirst-data/whosonfirst-data/issues/595 to address this issue.

**Comment by stepps00** [comment]
Regarding https://github.com/whosonfirst-data/whosonfirst-data/pull/559#pullrequestreview-13647203:

The commit for Djibouti contains 14 new features and 1 deprecated feature (https://github.com/whosonfirst-data/whosonfirst-data/pull/559/commits/7d90dd3d3da4b2822ba3a06e5e8ddfe379a3e107#diff-69d2c68828457ed8224d45b5efe2b6b2).
This is still off from what is shown on wikipedia ([here](https://en.wikipedia.org/wiki/Districts_of_Djibouti)), so https://github.com/whosonfirst-data/whosonfirst-data/issues/596 was opened to untangle this issue. At first glance, it looks like there were 14 districts in Djibouti, but there are now 20 districts.

**Comment by stepps00** [comment]
Regarding https://github.com/whosonfirst-data/whosonfirst-data/pull/559#pullrequestreview-13646709:

The three additional features are deprecated features... the counts of **new** features in the Ghana commit should match Statoids counts. However, the recent redistricting in Ghana which exploded the 110 `admin2` features into 216 `admin2` features has not been accounted for. I've opened https://github.com/whosonfirst-data/whosonfirst-data/issues/597 to address this issue.

**Comment by stepps00** [comment]
Regarding https://github.com/whosonfirst-data/whosonfirst-data/pull/559#pullrequestreview-13646168:

https://github.com/whosonfirst-data/whosonfirst-data/issues/598 was filed to track the new `admin2` divisions in Sierra Leone.

File: data/110/877/907/1/1108779071.geojson (hunk 2 above)
Comment (stepps00) [inline data/110/877/907/1/1108779071.geojson]: This (and another comment here) are being tracked through https://github.com/whosonfirst-data/whosonfirst-data/issues/598.

**Comment by stepps00** [comment]
Regarding https://github.com/whosonfirst-data/whosonfirst-data/pull/559#pullrequestreview-13647881:

I believe the Central African Republic features are correct and match Statoids. [The commit](https://github.com/whosonfirst-data/whosonfirst-data/pull/559/commits/c9250e59f2f85708bdf6c7dfb186d170a3cfdc9e) includes a deprecated feature.

**Comment by stepps00** [comment]
Additionally, issues https://github.com/whosonfirst-data/whosonfirst-data/issues/601 and https://github.com/whosonfirst-data/whosonfirst-data/issues/602 have been filed for Burundi and Vietnam.

This should take care of all comments in this PR.

**Pull request merged**